client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -115,6 +122,12 @@ export function WPOrderReviewLineItems( {
 									isDisabled={ isDisabled }
 									siteId={ siteId }
 									productSlug={ product.product_slug }
+									type={
+										isJetpackCheckout && experimentAssignment?.variationName === 'treatment'
+											? 'dropdown'
+											: 'buttons'
+									}
+									isLoading={ isJetpackCheckout && isLoadingExperimentAssignment }
```

Comment:
I tried separating the loading logic between here and the composite-checkout placeholder, but it still briefly flashed the radio buttons. `isLoading` is a temporary fix and can be removed once the test is complete.